<launch>
  <!-- params -->
  <param name="/simulator/scene_file" value="$(find pedsim_simulator)scenarios/static_group_nav.xml" type="string"/>

  <!-- other parameters -->
  <param name="/pedsim_simulator/max_robot_speed" value="2.0" type="double"/>
  <param name="/pedsim_simulator/robot_wait_time" value="3" type="double"/>
  <param name="/pedsim_simulator/teleop_flag" value="1.0" type="double"/>

  <!-- tf broadcaster for aligning frames -->
  <node name="simulator_tf_broadcaster" pkg="pedsim_simulator" type="dummy_transforms.py" respawn="false" output="log"
/>

  <!-- main simulator node -->
  <node name="simulator" pkg="pedsim_simulator" type="simulator" output="screen"/>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find pedsim_simulator)/rviz/social_contexts.rviz"/>


  <!-- planning node -->
  <!-- <node name="momo_node" pkg="momo" type="momo_ros.py" respawn="false" output="log"/> -->

  <!-- Keyboard teleoperation node -->
  <node name="teleop" pkg="pedsim_simulator" type="teleop" respawn="false" output="log" />

  <!-- pedsim to spencer converter -->
  <node name="pedsim_to_spencer" pkg="pedsim_simulator" type="pedsim_to_spencer" respawn="false" output="log" />
</launch>
